Vegeting/Global/UIComponent/GroupCategoryView.swift

```diff
@@ -50,20 +57,17 @@ final class GroupCategoryView: UIView {
         categoryCollectionView.constraint(to: self)
         categoryCollectionView.constraint(.heightAnchor, constant: 60)
     }
-    
-    private func configureCollectionView() {
-        categoryCollectionView.dataSource = self
-        categoryCollectionView.delegate = self
-    }
-    
+
     func changeCategoryList(with list: [String]) {
         categoryList = list
     }
     
     func getSelectedCategory() -> String? {
```

Comment:
p1;
get 발견~! 혹시 기회가 되시면 get을 삭제하고 selectedCategory로 해주세요.
```suggestion
    func selectedCategory() -> String? {
```

Reply:
[3c74305](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/164/commits/3c7430576252ae41d20941dc559a131c011a1428)에서 반영완료!